dask/dataframe/tests/test_groupby.py

```diff
@@ -20,12 +21,20 @@
     check_numeric_only_deprecation,
     tm,
 )
+from dask.dataframe._pyarrow import to_pyarrow_string
 from dask.dataframe.backends import grouper_dispatch
 from dask.dataframe.groupby import NUMERIC_ONLY_NOT_IMPLEMENTED
 from dask.dataframe.utils import assert_dask_graph, assert_eq, assert_max_deps
 from dask.utils import M
 from dask.utils_test import _check_warning, hlg_layer
 
+CONVERT_STRING = config.get("dataframe.convert_string")
+OBJECT_DTYPE = pd.StringDtype("pyarrow") if CONVERT_STRING else object
+CONVERT_STRING_FAIL_15 = pytest.mark.xfail(
+    CONVERT_STRING and not PANDAS_GT_200,
+    reason="not supported for string[pyarrow] below 2.0",
+)
```

Comment:
Should be safe to drop this now since we're only running the test suite with pyarrow strings turned on with `pandas>=2.0`